build.rs

```diff
@@ -207,14 +236,12 @@ fn main() {
         .expect("Couldn't write bindings!");
 
     /* OpenSSL Cryptography */
-    #[cfg(feature = "dynamic")]
-    use_system_ossl(&ossl_bindings);
-
-    #[cfg(not(feature = "dynamic"))]
-    println!("cargo:rerun-if-changed={}", ".git/modules/openssl/HEAD");
-
-    #[cfg(not(feature = "dynamic"))]
-    build_ossl(&ossl_bindings);
+    if features.dynamic {
+        use_system_ossl(&ossl_bindings);
+    } else {
+        println!("cargo:rerun-if-changed={}", ".git/modules/openssl/HEAD");
```

Comment:
afaik this is no longer relevant or it should point to the openssl directory instead of the submodule git tree?

Reply:
good catch